pyproject.toml

```diff
@@ -116,6 +121,7 @@ fixable = [
     "I"  # isort
 ]
 
-[tool.ruff.format]
-quote-style = "double"
-indent-style = "space"
+[tool.ruff.lint.per-file-ignores]
+"__init__.py" = ["F401"]
```

Comment:
As did previous reviewer, I would prefer defining `__all__` as established in python protocol for module describing what can be imported; but it's gonna work either way anyway